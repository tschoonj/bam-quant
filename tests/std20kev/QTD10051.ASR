File: C:\xrfdaten\2014KW30\chris\std20kev\std10051.spe
Start: 21.07.2014 18:28:45
Stop: 21.07.2014 18:30:25
BESSY: 297.9762667 297.9931969
IONI: 2.938236E-11 2.985629E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97663
Run: 100000
Dead: 2
ICR: 1570
OCR: 0
Accumulated BESSY: 59694.1637918
Accumulated Ioni: 5.94914766E-09
N Accumulation: 200
Normfactor:  3.4422720e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sb
exp_x 36.2603988647461 36.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  51   2      3.604     52449.       215.      10.52
