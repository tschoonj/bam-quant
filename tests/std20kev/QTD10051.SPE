$DATA:
0 2000
22952
14420
6302
1857
334
44
2
0
0
0
0
0
0
0
0
0
8
46
59
60
52
39
44
45
49
54
28
44
49
41
33
40
34
32
39
36
26
35
38
36
31
46
39
34
31
31
39
39
28
31
24
34
27
35
20
20
44
37
28
37
35
25
35
26
24
32
29
34
23
35
36
39
27
29
32
34
28
29
20
22
34
33
25
27
26
34
32
34
48
77
84
118
113
106
88
71
41
42
37
41
48
62
108
97
98
116
105
106
99
73
53
48
47
47
40
34
64
53
50
37
53
33
42
38
36
36
38
55
49
61
44
48
44
37
43
36
51
50
39
51
43
55
73
87
112
136
169
153
136
99
82
63
72
102
134
181
199
249
278
264
207
168
132
126
125
107
129
154
188
202
279
409
570
1076
1990
3308
5113
6673
7700
7464
6431
4641
2975
1836
1117
1081
1705
2847
4229
5617
6870
7620
7693
6903
5969
4947
3751
2801
1945
1554
1311
1421
1482
1566
1456
1236
845
569
328
213
166
140
195
274
276
343
333
286
225
150
86
91
88
137
212
266
364
331
317
286
224
169
137
75
46
24
27
11
15
6
9
9
5
12
4
11
11
9
14
7
8
6
9
13
7
16
14
8
5
8
11
12
9
6
5
2
13
10
13
8
9
8
8
11
9
11
11
15
13
15
12
9
8
7
10
9
12
12
11
8
19
8
10
14
11
6
4
8
6
11
15
14
9
15
5
8
5
12
10
9
22
12
12
12
20
13
19
18
10
16
18
12
12
16
8
14
13
9
12
5
13
15
16
18
12
12
10
11
13
13
18
10
16
18
9
16
16
12
9
11
12
19
12
5
11
13
15
19
12
11
17
15
7
18
17
9
11
12
12
23
21
14
12
15
17
9
12
16
10
13
7
12
17
12
12
24
18
17
12
11
19
18
17
19
15
11
13
14
17
13
12
18
11
20
17
8
10
15
8
8
18
14
14
17
21
19
12
16
9
15
15
11
16
16
13
16
15
14
14
21
15
16
16
19
10
21
16
12
11
12
14
13
14
23
11
16
15
21
16
18
19
10
15
12
16
11
20
14
14
14
17
17
19
16
16
16
14
15
14
12
21
18
17
10
20
14
10
16
19
13
12
18
15
16
20
26
21
6
16
19
13
17
11
12
15
18
19
16
13
20
12
14
22
15
20
17
18
18
17
12
18
20
15
14
19
19
14
11
12
17
14
16
17
19
13
15
11
16
17
7
17
11
22
16
11
8
12
18
19
16
16
15
19
11
14
12
18
15
18
18
16
18
16
25
18
10
17
17
20
11
13
14
17
10
10
21
12
9
13
17
13
10
15
9
13
12
18
12
14
12
12
17
16
14
15
15
8
12
13
11
16
11
13
15
14
17
13
18
16
15
11
14
9
15
12
16
17
9
16
14
14
15
13
18
13
13
16
15
15
10
18
8
21
17
16
15
15
17
19
14
16
18
15
10
16
12
14
16
8
15
16
13
21
17
9
14
12
11
16
16
16
14
16
9
9
8
12
9
13
11
16
12
12
12
20
17
15
13
10
13
10
5
14
13
15
10
13
16
10
9
9
14
13
16
8
12
15
15
15
10
9
14
13
16
8
17
18
14
11
8
11
8
17
11
18
10
15
11
13
14
15
13
12
12
9
8
19
12
12
14
13
7
12
11
15
13
13
17
13
10
13
10
16
15
8
20
18
9
8
9
8
12
11
18
7
19
5
14
12
13
12
10
12
11
11
9
8
10
23
11
6
21
14
7
8
6
19
15
11
14
13
14
14
31
28
36
31
51
44
56
53
58
37
42
37
22
34
23
8
9
9
8
8
7
5
2
5
5
4
1
4
2
5
3
5
2
5
4
2
1
5
2
5
3
4
4
4
5
3
4
2
5
3
3
0
2
6
1
7
4
2
2
2
3
1
5
5
2
2
3
1
2
4
4
2
3
2
7
2
2
8
1
3
9
4
7
6
6
10
4
6
4
6
5
5
12
12
7
9
5
8
9
7
14
12
11
13
17
11
11
9
10
9
11
8
5
5
7
12
12
6
12
8
1
5
10
10
6
1
10
7
8
10
14
9
10
7
9
11
12
13
15
14
11
19
18
15
13
16
21
12
17
16
15
15
17
19
22
12
16
18
15
16
26
18
14
29
21
23
26
26
24
22
28
36
31
32
25
34
28
28
35
31
32
33
40
32
33
45
39
35
34
54
52
44
44
53
59
54
45
57
63
67
69
76
85
98
107
119
146
169
194
204
248
259
309
340
355
380
373
367
374
368
321
318
304
333
291
240
266
232
179
183
118
124
94
91
53
52
50
19
15
12
14
7
10
4
3
2
0
0
3
1
0
1
0
1
2
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
2
0
1
2
1
4
5
2
3
5
7
4
6
5
2
9
3
5
3
8
9
7
9
5
6
8
6
6
2
2
7
2
1
1
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
1
3
1
4
2
1
0
1
3
2
0
0
1
1
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10051.spe
Start: 21.07.2014 18:28:45
Stop: 21.07.2014 18:30:25
BESSY: 297.9762667 297.9931969
IONI: 2.938236E-11 2.985629E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97663
Run: 100000
Dead: 2
ICR: 1570
OCR: 0
Accumulated BESSY: 59694.1637918
Accumulated Ioni: 5.94914766E-09
N Accumulation: 200
Normfactor:  3.4422720e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Sb
 
exp_x 36.2603988647461 36.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  51   2      3.604     52449.       215.      10.52
