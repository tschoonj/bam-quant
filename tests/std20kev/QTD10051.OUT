     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:52  บ   
     บ  Spectrum: QTD10051.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =    13.5   last change =   .00%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -48.0 ๑   100.0   บ     -48.0 ๑     2.4   บ   
     บ  GAIN (eV/ch)  บ     20.350 ๑   2.000   บ    20.350 ๑    .013   บ   
     บ det NOISE (eV) บ       62.3 ๑    40.0   บ      62.3 ๑    12.6   บ   
     บ  FANO factor   บ       .161 ๑    .050   บ      .161 ๑    .014   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Sb-L1                             25030. ๑    214.           บ   
     บ    L1M3        3.933    .48794       12213. ๑    105.    309.6  บ   
     บ              195.624    131.91         163.           6.95E-01  บ   
     บ    L1M2        3.886    .29919        7489. ๑     65.    286.0  บ   
     บ              193.314    131.30         163.           6.86E-01  บ   
     บ    L1N3        4.600    .19086        4777. ๑     43.    222.5  บ   
     บ              228.400    140.33         150.           7.95E-01  บ   
     บ    L1O3        4.697    .01134         284. ๑     12.    198.3  บ   
     บ              233.166    141.52         148.           8.06E-01  บ   
     บ    L1M5        4.167    .01066         267. ๑     13.     59.7  บ   
     บ              207.122    134.93         158.           7.36E-01  บ   
     บ                                                                 บ   
     บ  2 Sb-L2                             37198. ๑    242.           บ   
     บ    L2M4        3.844    .83845       31189. ๑    203.    226.0  บ   
     บ              191.250    130.75         164.           6.78E-01  บ   
     บ    L2N4        4.348    .13511        5026. ๑     35.    300.6  บ   
     บ              216.017    137.21         155.           7.63E-01  บ   
     บ    L2M1        3.437    .02090         777. ๑     14.     34.7  บ   
     บ              171.251    125.28         172.           5.83E-01  บ   
     บ    L2N1        4.229    .00554         206. ๑     13.    170.2  บ   
     บ              210.169    135.71         157.           7.45E-01  บ   
     บ                                                                 บ   
     บ  3 Sb-L3                             63194. ๑    259.           บ   
     บ    L3M5        3.604    .82997       52449. ๑    215.     10.5  บ   
     บ              179.457    127.55         169.           6.25E-01  บ   
     บ    L3N5        4.101    .13568        8574. ๑     37.    120.0  บ   
     บ              203.879    134.08         160.           7.25E-01  บ   
     บ    L3M1        3.189    .02600        1643. ๑     15.     13.0  บ   
     บ              159.064    121.83         177.           5.22E-01  บ   
     บ    L3N1        3.980    .00732         463. ๑     13.    302.2  บ   
     บ              197.933    132.52         162.           7.04E-01  บ   
     บ    L3O1        4.126    .00103          65. ๑     13.     58.7  บ   
     บ              205.108    134.40         159.           7.29E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                             215. ๑     25.           บ   
     บ    L2O4        7.476    .36552          79. ๑     15.      1.6  บ   
     บ              369.703    171.98         139.           9.47E-01  บ   
     บ    L2O1        7.208    .18844          40. ๑     13.      1.7  บ   
     บ              356.556    169.29         144.           9.41E-01  บ   
     บ    L2M3        7.743    .17726          38. ๑     12.      1.6  บ   
     บ              382.850    174.64         134.           9.52E-01  บ   
     บ    L3M5        7.707    .06400          14. ๑     12.      1.6  บ   
     บ              381.084    174.28         134.           9.52E-01  บ   
     บ    L3M4        7.975    .06207          13. ๑     11.      2.5  บ   
     บ              394.231    176.90         129.           9.56E-01  บ   
     บ    L3N5        8.209    .03636           8. ๑     11.      2.3  บ   
     บ              405.766    179.17         125.           9.60E-01  บ   
     บ    L3N4        7.952    .03610           8. ๑     11.      2.4  บ   
     บ              393.116    176.68         129.           9.56E-01  บ   
     บ    L3M1        8.477    .03526           8. ๑     11.      3.7  บ   
     บ              418.913    181.71         120.           9.63E-01  บ   
     บ    L3N1        8.220    .03501           8. ๑     11.      2.4  บ   
     บ              406.262    179.26         125.           9.60E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
